atest/acceptance/keywords/waiting.robot

```diff
@@ -52,6 +63,16 @@ Wait Until Page Does Not Contain Element
     ...    Custom Error ää ÖÖ
     ...    Wait Until Page Does Not Contain Element    content    0.1 seconds    Custom Error ää ÖÖ
 
+Wait Until Page Does Not Contain Element
+    [Documentation]    Tests also that format characters (e.g. %c) are handled correctly in error messages
+    Wait Until Page Does Not Contain Element    not_present    2 seconds    limit=1
+    Run Keyword And Expect Error
```

Comment:
1) Split test Wait Until Page Contains Element keyword is used to separate tests.
2) Limit argument is not used in on the error scenarios.
3) The current error test are pretty similar and do not in this context provide extra 
4) Write test with custom error 
5) Test for plural_or_not is missing

Reply:
Test cases added